scripts/tc/pkgs/u-boot

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-2-Clause
+
+U_BOOT_VER="2024.04"
+
+function internal_build_u_boot {
```

Comment:
That's great! You did exactly the same thing I did for busybox.

However, that requires one more thing: in order to build u-boot, the following additional packages were needed on my Ubuntu system:

- bison
- flex
- libssl-dev
- device-tree-compiler

The `build_toolchain` script takes care of installing all the necessary packages on the system for a smooth user experience. Can you add those 4 packages to the `pkg_list` in install_debian_family() in `scripts/tc/pkgs/install_pkgs` so that at least for Debian/Ubuntu we have that? I'll take care of the boring stuff to make that work on the other distros as well.

Reply:
ok，I'll fix that